orchestra/utils/project_properties.py

```diff
@@ -0,0 +1,22 @@
+from django.db.models import Count
+from orchestra.models import Project
+from orchestra.models import Task
+
+
+def completed_projects(projects):
+    """
+    Filters `projects` queryset to completed ones.
+
+    TODO(marcua): if we use this for business logic, we should also
+    ensure that the number of tasks equals the number of steps in
+    the workflow, which will be easier after @thisisdhaas implements
+    workflows-near-the-db.
+    """
```

Comment:
Also add a note that it would be cleaner to just add a Complete state to Projects.